### ISSUE TITLE
Text field for the current namespace isn't displayed at times

### ISSUE THREAD

**Issue by adietish** [redhat-developer/intellij-kubernetes#637]
I cannot replicate this (I'm on Mac), it was reported by @olkornii (using Linux) while doing the integration tests. It happens to him maybe 2 out of 3 times

**Steps:**
1. EXEC: In the resource tree: select the "Namespaces" category. Right click on it and pick "Set Current Namespace" (if you are using a kubernetes cluster. If it's OpenShift it's "Set Current Project"

**Result:**
A dialog shows up, that allows you to type the name of the namespace to use as current namespace. At times the dialog shows up without the text field (on Linux, not on Mac. Dont know about Windows)
<img width="298" alt="image" src="https://github.com/redhat-developer/intellij-kubernetes/assets/25126/886cf3b8-5a32-48b9-a82f-491053beeb10">

**Comment by adietish** [comment]
@olkornii: @JessicaJHee tried to replicate it on Linux but she couldn't. Things worked well for her. Maybe this was caused by your slow cluster?

**Comment by adietish** [comment]
Reopening since we are not sure if the change in #638 is fixing the issue. We could not replicate. PR still has value and was therefore merged.

**Comment by olkornii** [comment]
@adietish appears for me with minikube locally as well. So probably is not caused by slow cluster.
![Screenshot from 2023-07-24 10-02-13](https://github.com/redhat-developer/intellij-kubernetes/assets/60704619/4bfff77d-58c7-43d6-b20d-b5202c728755)

**Comment by olkornii** [comment]
@JessicaJHee Can I ask you to try again. For example, I had to try around 20 times to reproduce... 
Steps: Right click -> Set namespace -> Close

**Comment by JessicaJHee** [comment]
@olkornii I'm quite lost on why I'm not able to reproduce it :( I tried again with more than 20 times and it works as expected each time.

**Comment by olkornii** [comment]
@JessicaJHee Could you share your OS, Java version, cluster name and version please?

**Comment by JessicaJHee** [comment]
@olkornii Sure!

OS: Fedora 35
Java Version: openjdk 17.0.3 2022-04-19
Cluster: minikube, v1.29.0

**Comment by adietish** [comment]
@olkornii can you still reproduce this

**Comment by olkornii** [comment]
Hi @adietish and @mohitsuman, I'll check it tomorrow.

**Comment by olkornii** [comment]
@adietish 
Tried again with Fedora 39 CSB. Can not reproduce anymore.
But I noticed another strange behavior. Not sure if this is plugin related or not. Need one more person to check.

The steps are the same:
Right click "Namespaces" -> "Set Current Namespace" (HERE) -> close (HERE)

HERE: when clicked, fedora creates a new workspace for 1-2 seconds and deletes it. This is easy to see by the rectangles in the lower right corner of the screen.

**Comment by mohitsuman** [comment]
@adietish can you look at this for fedora OS ? If this is a new issue, let's track that separately and close this one.

**Comment by olkornii** [comment]
@adietish @mohitsuman While testing version 1.3.0, I noticed this bug again.

Unable to reproduce with IJ 2022.1 but easy to reproduce with 2024.1.
The steps are the same. Now this error is stable, the field appears only once and every next time there is no input field.

**Comment by mohitsuman** [comment]
@adietish can you please look at this ?

**Comment by adietish** [comment]
@mohitsuman, @olkornii: I dont have fedora, I can only check in MacOS

**Comment by adietish** [comment]
@olkornii: 
* what's your jdk when you face this issue? 
* what's your desktop (gnome?)? 
* what's your desktop theme?

**Comment by olkornii** [comment]
@adietish 
1. java --version: openjdk 11.0.22 2024-01-16
2. Gnome 45.5
3. Adwaita

**Comment by adietish** [comment]
@olkornii: you did not change `gradle.properties` and you are therefore running `IC-2022.1` (you can verify using Menu `About IntelliJ IDEA`), correct?

**Comment by olkornii** [comment]
I changed gradle.properties to 2024.1 for testing, as we discussed.

**Comment by adietish** [comment]
@olkornii: ok, so we can confirm that this occurs with latest `IC-2024.1` on Linux.

**Comment by adietish** [comment]
@olkornii, @sbouchet: if we can confirm that the above stacktrace is responsible for this, 

```java
2024-04-10 15:15:43,314 [ 299488] SEVERE - #com.redhat.devtools.intellij.kubernetes.actions.SetCurrentNamespaceAction - `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.redhat.devtools.intellij.kubernetes.actions.SetCurrentNamespaceAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.redhat.devtools.intellij.kubernetes]
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.redhat.devtools.intellij.kubernetes.actions.SetCurrentNamespaceAction' must override `getActionUpdateThread` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.redhat.devtools.intellij.kubernetes]
```

we'd have to bump min. supported IDEA version to `IC-2022.3`. See https://github.com/redhat-developer/intellij-openshift-connector/issues/738

**Comment by adietish** [comment]
@sbouchet, @olkornii: With [fix: default update thread for actions is EDT (#218)](https://github.com/redhat-developer/intellij-common/pull/219) in commons the stacktrace should not happen any more and this error should be fixed.
Can you please verify?

ps. intellij-common 1.9.4-SNAPSHOT wasn't published yet (timestamp should be of today 11th of April) due to issues on nexus. See https://issues.redhat.com/browse/NEXUS-815?focusedId=24517419&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel

**Comment by olkornii** [comment]
Hi @adietish , I've tried once more and still can reproduce. @sbouchet can I ask you to try as well?

**Comment by sbouchet** [comment]
will try to reproduce with latest snapshot.

**Comment by sbouchet** [comment]
still there: 
![Capture d’écran du 2024-04-15 12-00-03](https://github.com/redhat-developer/intellij-kubernetes/assets/600414/2aab40b9-5823-483c-9fb6-54b146ea9619)
